### ISSUE TITLE
cordova build android fails with cordova-plugin-admobpro

### ISSUE THREAD

**Issue by louisn** [pushbots/phonegap#39]
A build error occurs when cordova-plugin-admobpro is included with pushbots-cordova-plugin in the project.  The build output excerpt is shown below.  This can quickly be recreated by creating a new cordova project and adding the android platform plus the two plugins and then build.

:transformClassesWithDexForDebug
UNEXPECTED TOP-LEVEL EXCEPTION:
com.android.dex.DexException: Multiple dex files define Landroid/support/v4/accessibilityservice/AccessibilityServiceInfoCompat$AccessibilityServiceInfoVersionImpl;

**Comment by ihpalash** [comment]
Same also happening with me. Any solution found? I updated my android studio to the latest also the SDK

**Comment by louisn** [comment]
I found a work around.  Instead of using pushbots-cordova-plugin from npm, obtain one of the forked versions from github like this.

cordova plugin add https://github.com/dw-diamond/phonegap.git

It is very temperamental.  I could not simply retrofit my project, I had to create a new one and be very sure not to use the npm version because if even once the npm version gets added and built, the project seems to be corrupted and can't be repaired with plugin removal, clean, and addition processes.

We should pressure pushbots to update the npm version with the change found in the dw-diamond fork.
